backbone.js

```diff
@@ -1262,7 +1262,10 @@
     // Produces a DOM element to be assigned to your view. Exposed for
     // subclasses using an alternative DOM manipulation API.
     _createElement: function(tagName) {
-      return document.createElement(tagName);
+        namespace =  _.result(this, 'namespace');
```

Comment:
Sorry to confuse you, but `_.result(this, 'namespace')` should be in `#_ensureElement` to keep all our access in the same method. My [basic implementation](https://github.com/jashkenas/backbone/pull/3600#issuecomment-100942084) was just a stopgap in case we didn't decide to support namespaces.